FreeGlut doesn't support CGL, so I don't see why they're including the CGL
OpenGL headers.

https://sourceforge.net/p/freeglut/bugs/221

--- include/GL/freeglut_std.h.orig	2014-10-17 09:28:24.000000000 -0700
+++ include/GL/freeglut_std.h	2015-03-27 11:56:29.000000000 -0700
@@ -136,9 +136,6 @@
 #   include <EGL/egl.h>
 #   include <GLES/gl.h>
 #   include <GLES2/gl2.h>
-#elif __APPLE__
-#   include <OpenGL/gl.h>
-#   include <OpenGL/glu.h>
 #else
 #   include <GL/gl.h>
 #   include <GL/glu.h>
